### ISSUE TITLE
pyproject section warning

### ISSUE THREAD

**Issue by awgymer** [ofek/hatch-vcs#61]
I am just moving to `hatch-vcs` and after installing and adding minimal config to my `pyproject.toml` I am getting the following warning when running `hatch build`:

```
WARNING  pyproject.toml does not contain a tool.setuptools_scm section
```

`pyproject.toml`
```
[project]
...
dynamic = ["version"]
...

[tool.hatch.version]
source = "vcs"

[tool.hatch.build.hooks.vcs]
version-file = "mypackage/_version.py"
```

Versions:
```
hatch==1.9.1
hatch-vcs==0.4.0
hatchling==1.21.0
setuptools-scm==8.0.4
```

Is this warning expected? Is there a good way to mute it without just making a dummy section in my `pyproject.toml`?

**Comment by ofek** [comment]
@RonnyPfannschmidt Can you explain why this happens? I'd like to fix this.

**Comment by RonnyPfannschmidt** [comment]
I believe this happens because of virtualenv discover calling distuils which is patched

I think it's best if I disable the warning as it's not applicable These days

**Comment by ofek** [comment]
@awgymer Is this fixed if you upgrade Hatch?

**Comment by awgymer** [comment]
Upgrade to what, a dev version from github?

I am running the latest `PyPI` release of 1.9.1

**Comment by ofek** [comment]
Yeah that's what I meant. Okay, does it still produce that message if you run it again?

**Comment by awgymer** [comment]
Ah yes, it does indeed go away if I pip install the master branch from git

**Comment by ofek** [comment]
Oh sorry I meant I wanted to make sure that you are on the latest. Since master has not changed then I think you have answered my question and the message does go away the second time which means Ronny has correctly diagnosed the issue being virtual environment creation.

Now I have to find a way to remove the message.

**Comment by RonnyPfannschmidt** [comment]
It's a logging warning, I think you could "sabotage" the setuptools-scm logger for the time of the call requiring error or higher
